### PR TITLE
Add seo improvements

### LINKED ISSUE #20: Canonical URLs
We should add the canonical link tag to the website head. This is helpful for search engines.

### PR DESCRIPTION
**Pull request by mrdrogdrog**
This PR improves our SEObility and accessibility by applying some suggestions from lighthouse. 

Fixes #20

### CODE REVIEW

**Review by DerMolly — APPROVED** [review]

File: themes/vex-hugo/layouts/partials/banner.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
 {{- with $data.homepage.banner }}
 <!-- hero area -->
 <section class="d-lg-none d-flex align-items-center">
-	<img src="{{ .image | absURL }}" class="w-100" style="height: 300px;" alt="">
+	<img src="{{ .image | absURL }}" class="w-100" width="454" height="300" alt="">
```
Comment (ErikMichelson) [inline themes/vex-hugo/layouts/partials/banner.html:7]: An alt text (e.g. as parameter) would be great here.

File: themes/vex-hugo/layouts/partials/promo.html [hunk 2]
```diff
@@ -6,7 +6,7 @@
     <div class="container">
         <div class="row">
             <div class="col-md-6 text-center mb-5 mb-lg-0">
-                <img class="img-fluid" src="{{ .image | absURL }}" alt="">
+                <img class="img-fluid" src="{{ .image | absURL }}" height="177" width="178" alt="">
```
Comment (ErikMichelson) [inline themes/vex-hugo/layouts/partials/promo.html:9]: An alt text (e.g. as parameter) would be great here.

Comment (mrdrogdrog) [reply themes/vex-hugo/layouts/partials/promo.html]: done

File: themes/vex-hugo/layouts/partials/banner.html (hunk 1 above)
Comment (mrdrogdrog) [reply themes/vex-hugo/layouts/partials/banner.html]: done

**Review by ErikMichelson — APPROVED** [review]

**Pull request merged**